python/langsmith/schemas.py

```diff
@@ -370,7 +368,9 @@ class RunBase(BaseModel):
     tags: Optional[List[str]] = None
     """Tags for categorizing or annotating the run."""
 
-    attachments: Attachments = Field(default_factory=dict)
+    attachments: Union[Attachments, Dict[str, AttachmentInfo]] = Field(
```

Comment:
is this related to this pr?

Reply:
yeah need this for the changes in read_run